Telerik.JustMock/Expectations/Abstraction/IPrerequisite.cs

```diff
@@ -0,0 +1,35 @@
+/*
+ JustMock Lite
+ Copyright © 2019 Progress Software Corporation
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+namespace Telerik.JustMock.Expectations.Abstraction
+{
+    /// <summary>
+    /// Defines the prerequisite for mock setups.
+    /// </summary>
+    public interface IPrerequisite
+    {
+        /// <summary>
+        /// Determines wheter prerequisite is met
+        /// </summary>
+        bool IsMet { get; }
+
+        /// <summary>
+        /// Specifies a call should occur only after all of the given prerequisites have been called.
```

Comment:
This summary should be changed to:
Specifies that a call should occur only after all of the given prerequisites have been met.